underscore.js

```diff
@@ -647,6 +647,23 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    count = count || 1;
```

Comment:
I think the default = 1 makes sense. However do we want to mirror lodash
which makes anything less than 1 mean 1 chunk size?

E.g. _.chunk([1,2,3], 0) => [[1], [2], [3]]

On Tue, Jul 28, 2015 at 11:32 AM, Justin Ridgewell <notifications@github.com
> wrote:

> In underscore.js
> <https://github.com/jashkenas/underscore/pull/1919#discussion_r35661334>:
>
> > @@ -647,6 +647,23 @@
> >      return range;
> >    };
> >
> > +  // Split an **array** into several arrays containing **count** or less elements
> > +  // of initial array
> > +  _.chunk = function(array, count) {
> > +    count = count || 1;
>
> To support the count = undefined case and fix the count = 1 case, we
> should default to 0 here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/underscore/pull/1919/files#r35661334>.
>